checkov/gitlab_ci/image_referencer/provider.py

```diff
@@ -43,6 +44,8 @@ def extract_images_from_workflow(self) -> list[Image]:
                                         name=image_name,
                                         start_line=start_line,
                                         end_line=end_line,
+                                        related_resource_id=generate_resource_key_recursive(self.workflow_config, '',
```

Comment:
can use keyword arguments here, usually with 3+ arguments it is hard to understand which argument is which 😵